projects/bp-gallery/src/index.tsx

```diff
@@ -25,11 +25,13 @@ if (matomoUrl) {
   setupMatomo(matomoUrl);
 }
 
+export const root = document.getElementById('root');
+
 ReactDOM.render(
   <React.StrictMode>
     <BrowserRouter>{renderRoutes(routes)}</BrowserRouter>
   </React.StrictMode>,
-  document.getElementById('root')
+  root
```

Comment:
Um die fehlschlagenden Tests zu fixen hat es bei mir geholfen, diese Datei so zu lassen wie sie ist und stattdessen in den components direkt `document.getElementById('root')` zu nutzen

Reply:
I think waiting for #231 or #221 to be merged will work, too, don't you think?